py/genv/runners/ssh.py

```diff
@@ -0,0 +1,79 @@
+import asyncio
+import os
+from asyncio.subprocess import Process
+from typing import Dict, Optional, Iterable, List
+
+from .runner import Runner
+
+
+class SshRunner(Runner):
+    host_name: str
+    process_env: Dict[str, str]
+    __SSH_COMMAND_PREFIX = "ssh"
+    __SSH_TIMEOUT_PARAMETER = "-o ConnectTimeout={0}"
+
+    def __init__(self, host_name: str, timeout: Optional[int] = None, process_env: Optional[Dict] = None):
+        self.host_name = host_name
+        self.timeout = timeout
+        self.process_env = process_env
+
+    def name(self) -> str:
+        return self.host_name
+
+    async def _open_process(self, *args, stdin: bool = False, process_env: Optional[Dict[str, str]] = None,
+                            sudo: bool = False) -> (Process, str):
+        ssh_parameters = self.calc_ssh_params()
+        remote_command = self.calc_command_on_remote_machine(args, process_env, sudo)
+
+        return await asyncio.create_subprocess_exec(
```

Comment:
we return here only `Process` but you expect a tuple with the command, am I correct?
so I wonder if we tested this code